receiver/sqlserverreceiver/documentation.md

```diff
@@ -299,6 +299,86 @@ This metric is only available when the receiver is configured to directly connec
 | ---- | ----------- | ---------- |
 | {processes} | Gauge | Int |
 
+### sqlserver.query.execution_count
+
+Number of executions of the query
+
+This metric is only available when the receiver is configured to directly connect to SQL Server.
+
+| Unit | Metric Type | Value Type | Aggregation Temporality | Monotonic |
+| ---- | ----------- | ---------- | ----------------------- | --------- |
+| {executions} | Sum | Double | Unspecified | false |
```

Comment:
Should we change the metadata to correctly reflect the `Aggregation Temporality` and `Monotonic`?